app/experimenter/experiments/migrations/0212_alter_nimbusexperiment_languages_field_for_mobile_client.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+from experimenter.base.models import Language
+from experimenter.experiments.models import NimbusExperiment
+
+
+def update_languages_field_for_mobile_client(apps, schema_editor):
+    NimbusExperiments = apps.get_model("experiments", "NimbusExperiment")
+
+    for experiment in NimbusExperiments.objects.all():
```

Comment:
We don't want this to run for experiments that have already launched since then the database wouldn't agree with the recipe that was sent to users.  We should filter this to only draft 🙏